internal/shared/modules/mocks/bus_module_mock.go

```diff
@@ -0,0 +1,188 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
Why did we start committing mocked files now?

We did not do so before and shouldn't start doing it now.